community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/tracing/TransactionEvent.java

```diff
@@ -81,7 +81,7 @@ public void setReadOnly( boolean wasReadOnly )
 
     /**
      * Set the type of the transaction, as given by
-     * {@link org.neo4j.kernel.impl.api.KernelTransactionImplementation.TransactionType}.
+     * {@link org.neo4j.kernel.impl.api.KernelTransactionImplementation.TransactionWriteState}.
      */
     void setTransactionType( String transactionTypeName );
```

Comment:
Although, may I suggest changing method and parameter name to reflect the change from "transactionType" to "transactionWriteState"?